eos-phone-home

```diff
@@ -309,7 +313,8 @@ class PhoneHome(object):
             log.info("Getting variable: %s", name)
             var = getattr(self, '_get_' + name)()
 
-            if var is None or var == '':
+            if var in self.MANDATORY_STRINGS and \
+               (var is None or var == ''):
```

Comment:
If you're confident `var` is a string, you can spell this as `not var`: `None` and `""` are both false-y and all other strings are true-y ([source](https://docs.python.org/2/library/stdtypes.html#truth-value-testing)).

Reply:
:+1: